commands/tag-image.ts

```diff
@@ -37,11 +37,6 @@ export async function tagImage(context?: ImageNode): Promise<void> {
             imageName = defaultRegistryPath + '/' + imageName;
         }
 
-        const defaultRegistry = configOptions.get('defaultRegistry', '');
```

Comment:
Weren't you going to read the old setting, combine with the new one, and resave to the new one?

Reply:
Combining happens as part of `consolidateDefaultRegistrySettings`, which I intend to run on extension activation. 

Note: There is a change in that function -having the config setting update be async- that hasn't been pushed. I'm pushing another commit that fixes that, but also implements part 1 of the bug filed (ask to save setting on first push).